grpc/grpc
# Cronet/gRPC Core end-to-end test

This PR is created for mef@ troubleshooting gRPC and Cronet interfacing.

Instruction for setting up the test:

- Enter the test's directory:
`$ cd src/objective-c/tests`
- Install pods:
`$ CONFIG=opt pod install`
- Open the Tests workspace with Xcode:
`$ open Tests.xcworkspace`
- Select CoreCronetEnd2EndTests scheme
- Build
- In test navigator, run the individual tests under CoreCronetEnd2EndTests.

Expect some tests fail. In particular, test 'testSimpleRequest' is expected to fail at simple_request.c:207.
